web/packages/shared/package.json

```diff
@@ -13,5 +13,8 @@
     "@gravitational/design": "1.0.0",
     "ace-builds": "1.35.2",
     "highlight-words-core": "^1.2.2"
+  },
+  "devDependencies": {
+    "jest-fail-on-console": "^3.3.0"
```

Comment:
This is needed only because `setupTests.tsx` is in shared. I feel like it should be in build instead and it doesn't need the `.tsx` extension.

Reply:
Sure, I will move it to build.